api/app/jobs/fetch_video_thumbnails.rb

```diff
@@ -0,0 +1,10 @@
+# Job to fetch youtube and vimeo thumbnails in the background.
+class FetchVideoThumbnails < ApplicationJob
```

Comment:
Also, this class is too narrowly named, IMO. It's likely that we'll need to fetch thumbnails for other types of resources in the future. Let's call this FetchResourceThumbnails instead of FetchVideoThumbnails.